main.go

```diff
@@ -353,13 +354,14 @@ func deleteFileIfExists(requestId uint64, w http.ResponseWriter, r *http.Request
 }
 
 var paths = struct {
-	Video, VNC, Logs, Devtools, Download, Clipboard, File, Ping, Status, Error, WdHub, Welcome string
+	Video, VNC, Logs, Devtools, BiDi, Download, Clipboard, File, Ping, Status, Error, WdHub, Welcome string
 }{
 	Video:     "/video/",
 	VNC:       "/vnc/",
 	Logs:      "/logs/",
 	Devtools:  "/devtools/",
 	Download:  "/download/",
+	BiDi:      "/session/",
```

Comment:
Other paths are:
```
	Video:     "/video/",
	VNC:       "/vnc/",
	Logs:      "/logs/",
	Devtools:  "/devtools/",
	Download:  "/download/",
	Clipboard: "/clipboard/",
	Status:    "/status",
	File:      "/file",
```
So, path name matches the path

Why didn't you named "/session/" path as "Session"? Later there could be other cases, when we would need "/session/" path, but it would be called BiDi, which is confusing

If you want to emphasise "it is used by bidi", i would use a comment

Reply:
You are right,but I found that path `/session/` is already registered in `seleniumPaths`. So I will use it in order to not duplicate it